packages/amplify-cli-core/src/types.ts

```diff
@@ -20,19 +20,209 @@ export type $TSContext = {
    * @deprecated
    */
   print: IContextPrint;
-  migrationInfo: $TSAny;
+  migrationInfo: MigrationInfo;
   projectHasMobileHubResources: boolean;
   prompt: $TSAny;
   exeInfo: $TSAny;
-  input: $TSAny;
-  parameters: $TSAny;
-  usageData: $TSAny;
-  runtime: $TSAny;
+  input: $CommandLineInput;
+  parameters: ContextParameters;
+  usageData: IUsageData;
+  runtime: Runtime;
   pluginPlatform: IPluginPlatform;
-  newUserInfo?: $TSAny;
+  newUserInfo?: string;
```

Comment:
is this serialized json ?

Reply:
As best I could see, `newUserInfo` is the set to the awaited return value of `packages/amplify-provider-awscloudformation/src/setup-new-user.ts`'s `run` function.

<img width="459" alt="image" src="https://user-images.githubusercontent.com/18506410/218168180-15cea529-f90d-4e1b-ba05-b56afe06757f.png">

Reply:
see https://github.com/aws-amplify/amplify-cli/blob/66f65adf2280a433fff933ff40a7632bf21e804d/packages/amplify-provider-awscloudformation/src/setup-new-user.ts#L123 .
Seems that right name for this prop is `profileName` or `awsProfileName`.
See https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html .